cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -100,7 +100,7 @@ class ReporterCommand : CliktCommand(
 
     private val copyrightGarbageFile by option(
         "--copyright-garbage-file",
-        help = "A file containing garbage copyright statements entries which are to be ignored."
+        help = "A file containing copyright statements which are marked as garbage."
```

Comment:
Nit: Maybe better "A file containing garbage copyright statements that should be ignored."

Reply:
I deliberately removed the word "ignore", see the commit message.

Reply:
My point was that the sentence sounds as if the copyrights in the garbage file *already* are marked, instead of being used *for* marking. So maybe "which are *to be* marked as garbage"? But I'm also fine merging this as-is for now.